website/src/hugo/content/changelog.md

```diff
@@ -8,6 +8,98 @@
 Maintenance branches are merged before each new release. This change log is
 ordered chronologically, so each release contains all changes described below it.
 
+# v0.22.9 (2022-01-23)
+
+## What's Changed
+### http4s-core
```

Comment:
Unfortunately we have to change all these `###` to bullet points, otherwise laika balks IIRC.

Reply:
Oh, right.  Gack.

Reply:
This should probably be fixed upstream. It should be able to handle these duplicates by appending a counter to the anchor.

Reply:
Well, except the anchor's relative position is going to change over time, making it a not very useful anchor.

Reply:
Fair point. What it really needs to respect is the nesting I suppose. Asking that to be unique seems reasonable.